Jenkinsfile

```diff
@@ -1,68 +1,195 @@
-def NODE_NAME = "sw"
-def BRANCH_NAME_RELEASE = "release"
-def BRANCH_NAME_MASTER = "master"
+def SW_NODE = "windows-slave"
+def ECAT_NODE = "ecat-test"
+def ECAT_NODE_LOCK = "test_execution_lock_ecat"
+def CAN_NODE = "canopen-test"
+def CAN_NODE_LOCK = "test_execution_lock_can"
 def PYTHON_VERSIONS = ["3.6", "3.7", "3.8", "3.9"]
 def style_check = false
 
-node(NODE_NAME)
-{
-    deleteDir()
-    if (env.BRANCH_NAME == BRANCH_NAME_MASTER || env.BRANCH_NAME.contains(BRANCH_NAME_RELEASE))
-    {
-        stage("Checkout")
-        {
-            checkout scm
-        }
-        for (version in PYTHON_VERSIONS)
-        {   
-            stage("Python ${version}")
-            {
-                stage("Install environment ${version}")
-                {
-                    bat """
-                        py -${version} -m venv py${version}
-                        py${version}\\Scripts\\python.exe -m pip install -r requirements\\dev-requirements.txt
-                    """
-                }
-                if (!style_check) 
-                {
-                    stage("PEP8 style check")
-                    {
+
+pipeline {
+    agent none
+    stages {
+        stage('Build wheels and documentation') {
+            agent {
+                docker {
+                    label SW_NODE
+                    image 'ingeniacontainers.azurecr.io/ingenialink-builder'
+                }
+            }
+            stages {
+                stage('Clone repository') {
+                    steps {
                         bat """
-                            py${version}\\Scripts\\python.exe -m pycodestyle --first ingenialink/ --config=setup.cfg
+                            cd C:\\Users\\ContainerAdministrator
+                            git clone https://github.com/ingeniamc/ingenialink-python.git
+                            cd ingenialink-python
+                            git checkout ${env.GIT_COMMIT}
```

Comment:
I think the repository is already cloned, you only should do a checkout with "checkout scm"

Reply:
That is the first thing I tried, If we do It like that we get an error when installing the lib.

Reply:
Check the output of the Jenkins build #12 for the branch test-INGK-578.

Reply:
Ok, we will try to improve it in the future